### ISSUE TITLE
Installation broken with Cordova 7.0.0

### ISSUE THREAD

**Issue by yeldarby** [ionic-team/cordova-plugin-ios-keychain#21]
I get this error when I try to run the install step from `README.md`:

```
$ cordova plugin add https://github.com/driftyco/cordova-plugin-ios-keychain -d
No scripts found for hook "before_plugin_add".
Calling plugman.fetch on plugin "https://github.com/driftyco/cordova-plugin-ios-keychain"
saving
Running command: npm install https://github.com/driftyco/cordova-plugin-ios-keychain --save
Command finished with error code 235: npm install,https://github.com/driftyco/cordova-plugin-ios-keychain,--save
Error: Failed to fetch plugin https://github.com/driftyco/cordova-plugin-ios-keychain via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 235 Error output:
npm ERR! addLocal Could not install /var/folders/j7/gkwgxcp90gx2nqk08bht52xm0000gn/T/npm-9762-b7e4fc06/git-cache-23e5dc94/d9bbf389e7dad950452e73d72131573472f0a25f
npm ERR! code EISDIR
npm ERR! errno -21
npm ERR! syscall read
npm ERR! eisdir EISDIR: illegal operation on a directory, read
npm ERR! eisdir This is most likely not a problem with npm itself
npm ERR! eisdir and is related to npm not being able to find a package.json in
npm ERR! eisdir a package you are trying to install.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/yeldarb/.npm/_logs/2017-05-11T20_10_01_038Z-debug.log
```

**Comment by netmikey** [comment]
I have the same issue trying to install the exoplayer plugin from Github on macOS:

```
$ ../node_modules/.bin/cordova plugin add https://github.com/frontyard/cordova-plugin-exoplayer
Error: Failed to fetch plugin https://github.com/frontyard/cordova-plugin-exoplayer via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 235 Error output:
npm ERR! addLocal Could not install /var/folders/yq/t6hfv8453z5693m7zt4m0qsh0000gn/T/npm-5152-64b122ae/git-cache-65041863/eb052c879ecfc600ddd09510c72f491a821a5fb7
npm ERR! Darwin 16.5.0
npm ERR! argv "/opt/local/bin/node" "/opt/local/bin/npm" "install" "https://github.com/frontyard/cordova-plugin-exoplayer" "--save"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.8
npm ERR! code EISDIR
npm ERR! errno -21
npm ERR! syscall read

npm ERR! eisdir EISDIR: illegal operation on a directory, read
npm ERR! eisdir This is most likely not a problem with npm itself
npm ERR! eisdir and is related to npm not being able to find a package.json in
npm ERR! eisdir a package you are trying to install.
```

**Comment by yeldarby** [comment]
I hope the plugin maintainer doesn't mind but I was able to fix this temporarily by forking the repo, publishing it to npm, and installing it from there.

**Comment by netmikey** [comment]
Looks like this is coming from this issue: https://issues.apache.org/jira/browse/CB-12680

TL;DR: The plugin needs to include a `package.json` to be fetchable from git.

**Comment by wilkesreid** [comment]
I just submitted a [pull request](https://github.com/driftyco/cordova-plugin-ios-keychain/pull/23) to just add a generic package.json to fix this.

**Comment by samuelcastro** [comment]
Any updates here, I'm still getting the same issue?

**Comment by bg-wilkesreid** [comment]
We are waiting on @mlynch to merge the pull request that will solve this.

**Comment by mlynch** [comment]
Merged some fixes to this, cheers
